### PR TITLE
ci: fix conditions in publishing workflow

### PR DESCRIPTION
**Pull request by ligurio**
NO_CHANGELOG=ci
NO_DOC=ci
NO_TEST=ci

### CODE REVIEW

File: .github/workflows/publish-module-api-doc.yaml [hunk 1]
```diff
@@ -3,9 +3,17 @@ name: Publish module API documentation
 on:
   push:
     branches: [master]
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+  workflow_dispatch:
 
 jobs:
   publish-api-doc:
+    # Run on push to the 'master' or on pull request if the 'full-ci' label is
```
Comment (ylobankov) [inline .github/workflows/publish-module-api-doc.yaml:12]: Could you please add `branch` after `'master'`?

Comment (ylobankov) [reply .github/workflows/publish-module-api-doc.yaml]: Looks like fixed.

**Comment by ylobankov** [comment]
Please resolve conflicts.

**Review by igormunkin — APPROVED** [review]
@ligurio, thanks for the patch! LGTM, as trivial.

**Review by igormunkin — CHANGES_REQUESTED** [review]
@ligurio, thanks for the patch! Please mention the previous commit and the fixup as the reason the condition of the workflow is changed. Also, describe the changes a little in the commit message. Finally, rebase your changes on the bleeding master to resolve the conficts.

By the way, what about the concurrency section used in all other workflows? @ylobankov, do we need it here too?

**Comment by ylobankov** [comment]
> @ligurio, thanks for the patch! Please mention the previous commit and the fixup as the reason the condition of the workflow is changed. Also, describe the changes a little in the commit message. Finally, rebase your changes on the bleeding master to resolve the conficts.
> 
> By the way, what about the concurrency section used in all other workflows? @ylobankov, do we need it here too?

This workflow is not so heavy, but we can add it for consistency with others.

**Review by ylobankov — APPROVED** [review]
LGTM, please fix Igor's comments.

**Review by igormunkin — APPROVED** [review]
@ligurio, thanks for the fixes! LGTM.

**Pull request merged**